node/src/chain/header_validation.rs

```diff
@@ -99,6 +106,32 @@ impl<'a, DB: database::DB> Validator<'a, DB> {
             Ok(())
         })?;
 
+        // Verify seed field
+        if candidate_block.height > 1 {
```

Comment:
Why do we start with height 2?

Reply:
Because `seed` in Genesis block is always empty.

Reply:
Corrected